### DISCUSSION TITLE
All poetry commands failing: [Errno 2] No such file or directory: 'python'

### DISCUSSION THREAD

**Discussion by tyteen4a03** [python-poetry/poetry#5318] [Q&A]
Hi, I just tried to install poetry 1.1.13 on MacOS 12.3 (Intel Mac with Homebrew Python 3.10), but almost all of the commands are failing with the below error:

```
  FileNotFoundError

  [Errno 2] No such file or directory: 'python'

  at /usr/local/Cellar/python@3.10/3.10.2/Frameworks/Python.framework/Versions/3.10/lib/python3.10/subprocess.py:1842 in _execute_child
      1838│                     else:
      1839│                         err_filename = orig_executable
      1840│                     if errno_num != 0:
      1841│                         err_msg = os.strerror(errno_num)
    → 1842│                     raise child_exception_type(errno_num, err_msg, err_filename)
      1843│                 raise child_exception_type(err_msg)
      1844│
      1845│
      1846│         def _handle_exitstatus(self, sts,
      ```

**Comment by cansozbir** [comment]
I have the same error on one of my docker projects. Did you find a workaround @tyteen4a03 ? 
Currently, I'm trying to figure out how can I specify a python version on a docker file, so I can try it with python 3.8.  Have you tried downgrading your python, if so how did it result?

Edit: downgrading python version solved my issue.
